### ISSUE TITLE
"no-member" not triggered for C-extension member but "--unsafe-load-any-extension=y"

### ISSUE THREAD

**Issue by buhtz** [pylint-dev/pylint#9541]
### Question

I do know about the exceptional behavior of PyLint when it comes to c-extensions. I thought I had workaround that but it seems I still misunderstand something.

I expect a `no-membe (E1101)` error at this line ([original code](https://github.com/bit-team/backintime/blob/d33f35e10b4c3027b3b12881152c157eb1b2cc8c/qt/settingsdialog.py#L2271-L2275)):

        self.colorRed.setColor(QPalette.WindowText, QColor(205, 0, 0))

The class `QPalette` is imported correct from `PyQt6`. But the member `WindowText` do not exit. Correct would be `QPalette.ColorRole.WindowText`. But PyLint do not see this error.

This is the pylint call I do use to test this:

```
pylint --disable=E0401,C0301,C0116,W0108,C0209,R0902,R0914,C0411,C0103,W0212,R0915,W0613,C0115,C3001,C0302,W0511,W0702,R1725,C0114,C0415,W0612,R0912,R1705 --extension-pkg-whitelist=PyQt6,PyQt6.QtCore --additional-builtins=_,ngettext --max-line-length=79 --good-names=idx,fp --unsafe-load-any-extension=y settingsdialog.py
```

### Documentation for future user

Don't know because I do not understand the underlying problem.

### Additional context

https://github.com/bit-team/backintime/blob/dev/qt/settingsdialog.py#L2271-L2275
